.github/workflows/main.yml

```diff
@@ -149,6 +149,31 @@ jobs:
         timeout-minutes: 4
         run: yarn test:fastboot ${{ matrix.scenario }}
 
+  basic-tests-safari:
+    needs: [lint]
+    strategy:
+      matrix:
+        os: [macos-latest]
```

Comment:
this should look like the same strategy as the basic-tests-ie11 but with `runs-on` set to `macos-latest` and with the env set to enable Safari.

Reply:
great!  Another roadblock it seems.  OpenGL and OpenCL were deprecated in Catalina.  GH Actions doesn't support Mojave anymore.  Looking at alternative solutions....